### PR TITLE
Fix for refresh hardware (LMS) content param - port,   and update (MySensors) hardware  

### PR DESCRIPTION
**Pull request by Eorciu**
1. I commented line which is responsible for no port refresh in hardware content params "Logitech Media Server".

image

2. Fix error when update or refresh some type of hardware (for example "Mysensor Gataway for Lan", becouse "indexof" is not function, correct name is "indexOf"

### CODE REVIEW

File: www/app/HardwareController.js [hunk 1]
```diff
@@ -5609,7 +5609,7 @@ define(['app'], function (app) {
 				$("#hardwarecontent #divserial").hide();
 				$("#hardwarecontent #divremote").show();
 				$("#hardwarecontent #divlogin").show();
-				$("#hardwarecontent #hardwareparamsremote #tcpport").val(9000);
+//				$("#hardwarecontent #hardwareparamsremote #tcpport").val(9000);
```
Comment (gizmocuz) [inline www/app/HardwareController.js:5612]: I think this line should be

$("#hardwarecontent #divremote #tcpport").val(9000);

could you check this ?

Comment (Eorciu) [reply www/app/HardwareController.js:5612]: I double checked , unfortunately, Your line does not work.

Comment (gizmocuz) [reply www/app/HardwareController.js:5612]: Strange as this is how it is used on the other location in this file. It should not crash
Your patch just does not set the value anymore...

Comment (gizmocuz) [reply www/app/HardwareController.js:5612]: I will merge this patch and have a closer look

**Comment by gizmocuz** [comment]
Thanks!

File: www/app/HardwareController.js (hunk 1 above)
Comment (gizmocuz) [reply www/app/HardwareController.js:5612]: Checked and tested, and it works like it should
https://github.com/domoticz/domoticz/commit/1c7411dc561901d9ca430254e8db4c6be731fefb

**Pull request merged**